Base/ParticlePairDiffHistos.hpp

```diff
@@ -39,6 +39,8 @@ class ParticlePairDiffHistos : public Histograms
   template <AnalysisConfiguration::RapidityPseudoRapidity r, typename ParticleType1, typename ParticleType2>
   float getDeltaPhi(ParticleType1& particle1, ParticleType2& particle2);
   template <typename ParticleType1, typename ParticleType2>
+  void getInvMass(ParticleType1& particle1, ParticleType2& particle2, double& invMass);
```

Comment:
Don`t pass the `invMass` as parameter
Return the calculated invariant mass and change the type of the function accordingly
Do we need double precision for the invariant mass?